docs/translations/translations.json

```diff
@@ -292,6 +292,7 @@
     "/customization/default-theme": "Default Theme",
     "/customization/how-to-customize": "How To Customize",
     "/customization/color": "Color",
+    "/customization/styling-engine": "Styling Engine",
```

Comment:
The package is named `styled-engine` but the page refers to `styling-engine`? Are these different things or can we use the same term unless there's a grammatical problem I'm missing.

Reply:
Will rename to Styled Engine 👍